public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
+User-agent: *
+Disallow:
```

Comment:
- [ ] All the boilerplate  that got created by automatically (texts, images, and styles) need to be removed(stuff such as on public folder etc)

Reply:
Done